src/CAREUI/misc/PaginatedList.tsx

```diff
@@ -97,6 +102,18 @@ const WhenLoading = <TItem extends object>(props: WhenEmptyProps) => {
 
 PaginatedList.WhenLoading = WhenLoading;
 
+const Display = <TItem extends object>(props: DisplayProps) => {
+  const { loading } = useContextualized<TItem>();
+
+  if (loading) {
+    return null;
+  }
+
+  return <div className={props.className}>{props.children}</div>;
+};
+
+PaginatedList.Display = Display;
+
```

Comment:
This (`PaginatedList.Display` could be confusing, especially the name doesn't convey any meaning of which to use when).

Instead add a prop to `PaginatedList.Items` component: `hideIfLoading` (boolean), that'll achieve this same functionality.

Reply:
Got it, I added the logic in `Items`. It will return null if its loading. Props aren't required we can get loading state from `useContextualized`